interface/modules/custom_modules/oe-module-comlink-telehealth/src/Controller/TeleHealthCalendarController.php

```diff
@@ -182,12 +190,29 @@ public function renderAppointmentsLaunchSessionButton(AppointmentRenderEvent $ev
             return;
         }
         // don't show the launch button for a complete status
-        if ($this->apptService->isCheckOutStatus($row['pc_apptstatus'])) {
+        if ($this->getAppointmentService()->isCheckOutStatus($row['pc_apptstatus'])) {
+            echo "<button class='mt-2 btn btn-disabled' disabled><i class='fa fa-video m-2'></i>"
+                . xlt("TeleHealth Session Ended") . "</button>";
+            echo "<p>" . xlt("Session has been completed.") . " "
+                . xl("Change the appointment status in order to launch this session again.") . "</p>";
```

Comment:
xl to xlt (prob the only comment i will have in this last code review :) )

Reply:
Fixed